[2024-10-16T01:01:58.906+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-10-16T00:55:00+00:00 [queued]>
[2024-10-16T01:01:58.910+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-10-16T00:55:00+00:00 [queued]>
[2024-10-16T01:01:58.910+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T01:01:58.910+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-10-16T01:01:58.910+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T01:01:58.915+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2024-10-16 00:55:00+00:00
[2024-10-16T01:01:58.917+0000] {standard_task_runner.py:55} INFO - Started process 369 to run task
[2024-10-16T01:01:58.919+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2024-10-16T00:55:00+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpeasc38ep']
[2024-10-16T01:01:58.919+0000] {standard_task_runner.py:83} INFO - Job 88: Subtask create_table
[2024-10-16T01:01:58.943+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2024-10-16T00:55:00+00:00 [running]> on host dcdd4c71b617
[2024-10-16T01:01:58.967+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-10-16T00:55:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-10-16T00:55:00+00:00
[2024-10-16T01:01:58.971+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-10-16T01:01:58.972+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2024-10-16T01:01:58.973+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2024-10-16T01:01:58.976+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20241016T005500, start_date=20241016T010158, end_date=20241016T010158
[2024-10-16T01:01:59.016+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-10-16T01:01:59.028+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
